### DISCUSSION TITLE
用润的方法来解决所有问题真是一个可爱的想法

### DISCUSSION THREAD

**Discussion by hnnw** [The-Run-Philosophy-Organization/run#389] [General]
润的地方出现问题再润一个就是了，一定会有一个完美的地方可以润

**Comment by DabanC** [comment]
他们还有个很可爱的误解，他们似乎认为很多人会骂他们，很多人会酸他们。其实他们没那么重要，没有人关注他们怎么弄，有大把的合法的移民中介，根本灭必要搞的像地下党接头一样，无非就是啥本事也没有，想以政治庇护的方式移民嘛，有啥不能说的啊。祝福你们，希望应润尽润，最好带走一两个亿，功德无量🙏。

**Reply by DabanC** [reply]
当时颜宁走也没怎么样啊，真以为缺了谁不行吗，要润就润啊，在网上嚎有什么用，应润尽润，不要逼逼直接走，正规的走不了还能去墨西哥走线，快走快走快走

**Comment by the-run-philosophy** [comment]
这可不同，有些地方是允许人民自主自权自治的。
有些地方，则绝对不允许（包括你玩什么游戏看什么电视哦）。
你这种逻辑滑坡也算是贻笑大方了....
建议你看看什么叫布尔什维克。什么叫5000年文明的伟大延续。
什么叫“法国大革命”。以及为什么中国永远也无法有法国人的觉悟。

**Reply by hnnw** [reply]
一个地方的优势必定牺牲了一些劣势，而幻想一个完美的地方，这种逻辑真可爱。建议你学习学习什么是辩证唯物主义。中国人确实永远无法拥有法国人的觉悟（指双手举白旗）

**Reply by littleANT800** [reply]
> 一个地方的优势必定牺牲了一些劣势，而幻想一个完美的地方，这种逻辑真可爱。建议你学习学习什么是辩证唯物主义。中国人确实永远无法拥有法国人的觉悟（指双手举白旗）

我觉得没有人会认为一个地方是完美的，是可以解决一切问题的，重要的是自己心里想要什么，而这个东西哪里能给。

**Reply by hnnw** [reply]
> > 一个地方的优势必定牺牲了一些劣势，而幻想一个完美的地方，这种逻辑真可爱。建议你学习学习什么是辩证唯物主义。中国人确实永远无法拥有法国人的觉悟（指双手举白旗）
> 
> 我觉得没有人会认为一个地方是完美的，是可以解决一切问题的，重要的是自己心里想要什么，而这个东西哪里能给。
你这个逻辑倒是没毛病，如果是为了更好的工作机会、更好的收入、更好的生活，靠自己的技术、实力去润倒也没啥问题。看层主的回答”有些地方是允许人民自主自权自治的。“，我可不认为他知道自己心里想要啥，也不认为有技术和实力润。

**Reply by xyzhou-1** [reply]
"人民自主自权自治".自由是块砖,哪里需要往哪搬.
100万人敲1000次钟就完事了,羡慕死了.
国内因为防疫不利过失死亡了一些人,都要被你们喷死了.100万人确实只是个数字罢了.

想从发展中国家跑到发达国家合理,合法.还要扯些哲学概念就搞笑了.

**Reply by the-run-philosophy** [reply]
就经济而言，上海gdp已经超过发达国家了。按你这么说，上海不会有一个人想润。
你自己看看经济和政治如何能够分开看待吧。

**Comment by Evcklli** [comment]
支，润只是必要不充分条件，这么喜欢迫害鼠鼠是吧

**Comment by MarionNoel** [comment]
出国与否其实不重要。润太祖可是特大人命事故入狱的局长公子，贪二代，有钱的大老板。他大发善心，关爱新一代it耗材，安排咱们世世代代跟着他，给他做贡献。
https://github.com/MarionNoel/juzhanggongzi/discussions/
